ElementX/Sources/Screens/RoomScreen/RoomScreenModels.swift

```diff
@@ -172,10 +172,13 @@ struct RoomScreenViewState: BindableState {
     var isPinningEnabled = false
     var lastScrollDirection: ScrollDirection?
     
+    // The `pinnedEventIDs` is used only to determine if an item can be pinned or not since it does not depend on the pinned events timeline, but on the room info update which is faster
+    var pinnedEventIDs: Set<String> = []
+    // This is used to controler the banner
     var pinnedEventsState = PinnedEventsState()
```

Comment:
```suggestion
    // This is used to control the banner
    var pinnedEventsState = PinnedEventsState()
```

Although I would just delete that comment ¯\_(ツ)_/¯